Dockerfile

```diff
@@ -1,6 +1,22 @@
-FROM debian:stable-slim
+FROM golang:1.14 AS build
 
-RUN apt-get update && \
-    apt-get -y install ca-certificates
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
 
-ADD gopath/bin/tm /usr/bin/tm
+WORKDIR /tm
+
+COPY go.mod go.sum /tm/
+RUN go mod download
+
+ARG GIT_TAG
+ENV GIT_TAG=${GIT_TAG:-unknown}
+
+COPY . .
+RUN make install
+
+FROM scratch
+
+COPY --from=build /go/bin/tm /bin/tm
+
+ENTRYPOINT ["/bin/tm"]
```

Comment:
probably  a breaking change if this image is used somewhere in the internal pipeline. @tzununbekov should i remove the this instructions?

Reply:
iirc, we made this image mainly for Gitlab integration, eg: 
https://gitlab.com/sebgoa/kdemo/-/blob/master/.gitlab-ci.yml#L17

- I'd consider this scenario as main thing to care about building this image

Reply:
that was my concern. i've removed the `ENTRYPOINT` statement to avoid any issue that may arise as a result of the change